package/MDAnalysis/lib/util.py

```diff
@@ -1689,6 +1690,25 @@ def blocks_of(a, n, m):
 
     return np.lib.stride_tricks.as_strided(a, new_shape, new_strides)
 
+
+def group_consecutive_integers(arr):
+    """Split an array of integers into a list of consecutive sequences.
+
+    Parameters
+    ----------
+    arr: :class:`numpy.ndarray`
+
+    Returns
+    -------
+    list of :class:`numpy.ndarray`
+
+    Examples
+    >>> arr = np.array([ 2,  3,  4,  7,  8,  9, 10, 11, 15, 16])
+    >>> group_consecutive_integers(arr)
+    [array([2, 3, 4]), array([ 7,  8,  9, 10, 11]), array([15, 16])]
+    """
+    return np.split(arr, np.where(np.ediff1d(arr)-1)[0]+1)
```

Comment:
pep 8 spaces...

Reply:
PEP8 is ambiguous on spaces around operators; I typically just go with what makes the most sense/readability to me. 

<img width="500" alt="Screenshot 2020-06-06 at 5 56 52 AM" src="https://user-images.githubusercontent.com/31115101/83917759-c5a67e80-a7ba-11ea-909d-f7998f6be1a7.png">

Is the below easier to read?

```python
return np.split(arr, np.where(np.ediff1d(arr) - 1)[0] + 1)
```

Reply:
It's mildly confusing either way; maybe a middle ground:
```python
return np.split(arr, np.where(np.ediff1d(arr)-1)[0] + 1)
```